### ISSUE TITLE
Bundle: Request form: Screen reader output 

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2913]
### Story
As an assistive technology enabled uder, I want the screen reader output on the "Other employees from your organization" and "Add'l details" sections of the domain request form to be properly formatted and accessible, so that I can navigate and complete the form efficiently and without confusion.

### Acceptance Criteria
- [x] #2914
- [x] #2915
- [x] #2918
- [x] On the Other employees from your organization page, move the _Required fields are marked with an asterisk (*)._ below _We typically don’t reach out to these employees, but if contact is necessary, our practice is to coordinate with you first._
- [x] #2921
- [x] #2922
- [x] #2923 

### Additional Context

Reminder: no unit tests are required for screen reader output
How to view screen reader output using ANDI
- [Download ANDI](https://www.ssa.gov/accessibility/andi/help/install.html)
- Open ANDI
- Navigate to focusable elements
- Use the ⏪  ⏩ buttons to navigate through the page or hover over the element you want to inspect
- Use this handy [ANDI key](https://docs.google.com/document/d/1WTFBFvycmi8DjjxAVHjRjTIuq-QYkwCSNsN1XVrcs54/edit) to understand how the ANDI output colors map to code `attributes` and `<elements>`.

![Screenshot 2024-10-08 at 9 00 09 AM](https://github.com/user-attachments/assets/981bef3e-6c64-423a-966b-7dc98a9bab58)

NOTE: No unit tests are required for screen reader output.
### Issue Links
- Caused by #2866 
- Relates to #2924 

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
1.3.1 - Info and Relationships (Critical)
3.3.2 - Labels or Instructions (Critical)
4.1.2 - Name, Role, Value (Critical)
**Critical Failure if Not Met?:**  Yes
**Prioritization:** Blocker
**Reason:** This story addresses critical accessibility issues that affect users relying on assistive technologies. In this context, the reader output is not properly formatted and does not provide effective form navigation and completion. Mis-formatted outputs can lead to confusion, resulting in a negative user experience and potential failure to submit necessary information.

**Comment by AnnaGingle** [comment]
@abroddrick has reviewed and approved thid ticket.
